lib/execution/runner.js

```diff
@@ -238,7 +238,7 @@ class Runner {
 
     const results = [];
     for (const query of queries) {
-      results.push(await this.query(query));
+      results.push(await this.queryArray([query]));
     }
```

Comment:
what's the benefit of calling queryArray with single element?

Reply:
This ensures that the statements producer is executed. Maybe we should move the statements producer handling to `runner.query()`.